packaging/debian/control

```diff
@@ -20,6 +20,7 @@ Build-Depends: debhelper (>= 9),
     uuid-dev,
     libsystemd-dev,
     liblz4-dev,
+    libcurl-dev,
```

Comment:
this will break all package builds as libcurl-dev doesn't exist

Could you please update its entry in known_projects: https://github.com/zeromq/zproject/blob/master/zproject_known_projects.xml#L94

It should use one of those:

https://packages.debian.org/source/sid/curl

probably libcurl4-gnutls-dev, as the OpenSSL license _might_ be incompatible with the MPL2

Reply:
actually there are provides, so it might work - I'll need to add disambiguation on OBS

Reply:
no it breaks:

```
$ sudo apt install libcurl-dev
[sudo] password for luca: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package libcurl-dev is a virtual package provided by:
  libcurl4-openssl-dev 7.52.1-5+deb9u8
  libcurl4-nss-dev 7.52.1-5+deb9u8
  libcurl4-gnutls-dev 7.52.1-5+deb9u8
You should explicitly select one to install.

E: Package 'libcurl-dev' has no installation candidate
```

Reply:
I'm not sure how to fix this...

Reply:
it is already under known projects.

Reply:
probably the right one is libcurl4-openssl-dev, but this is generated file, how do I make it use libcurl4-openssl-dev?